instrumentation/netty/netty-4.1/javaagent/src/test/groovy/Netty41ServerTest.groovy

```diff
@@ -71,15 +71,17 @@ class Netty41ServerTest extends HttpServerTest<EventLoopGroup> implements AgentT
                       response = new DefaultFullHttpResponse(HTTP_1_1, HttpResponseStatus.valueOf(endpoint.status), content)
                       break
                     case INDEXED_CHILD:
+                      content = Unpooled.EMPTY_BUFFER
```

Comment:
@trustin @minwoox @ikhoon OkHttp client worked fine with the old code but Armeria seems to require the content length to be set. Is it expected?

Reply:
Ah to clarify the behavior is that Armeria client times out (timeout is set to 1 minute). Just doesn't seem to treat the response as complete